pkg/morph/client/static.go

```diff
@@ -168,11 +168,12 @@ func (s StaticClient) TestInvoke(prm TestInvokePrm) ([]stackitem.Item, error) {
 
 // TestInvokeIterator calls TestInvokeIterator method of [Client] with static
 // internal script hash.
-func (s StaticClient) TestInvokeIterator(prm TestInvokePrm) ([]stackitem.Item, error) {
```

Comment:
Is this (`TestInvokePrm`) type used anywhere else?

Reply:
yes, originally it is for `TestInvoke` without iteration, just was reused